includes/push-ui.php

```diff
@@ -498,13 +532,23 @@ function menu_content() {
 								 */
 								$as_draft = apply_filters( 'dt_allow_as_draft_distribute', $as_draft, $connection, $post );
 							?>
-							<button class="syndicate-button"><?php esc_html_e( 'Distribute', 'distributor' ); ?></button> <?php if ( $as_draft ) : ?><label class="as-draft" for="dt-as-draft"><input type="checkbox" id="dt-as-draft" checked> <?php esc_html_e( 'As draft', 'distributor' ); ?></label><?php endif; ?>
+							<button class="syndicate-button"><?php esc_html_e( 'Distribute', 'distributor' ); ?></button>
+							<?php if ( $as_draft ) : ?>
+								<label class="as-draft" for="dt-as-draft"><input type="checkbox" id="dt-as-draft" checked> <?php esc_html_e( 'As draft', 'distributor' ); ?></label>
+							<?php endif; ?>
 						</div>
 					</div>
 
 					<div class="messages">
 						<div class="dt-success">
-							<?php esc_html_e( 'Post successfully distributed.', 'distributor' ); ?>
+							<?php
+							/**
+							 * Filter distribution success message
+							 *
+							 * @param string Success message
+							 */
+							echo esc_html( apply_filters( 'dt_successfully_distributed_message', __( 'Post successfully distributed.', 'distributor' ) ) );
```

Comment:
this filter is related to the success message, can you add a sentence describing how it can be used/your use case

Reply:
In case when we are terminating 'push' function execution, it would be more descriptive to show another status message instead of "Post successfully distributed". For example: "Scheduled action to distribute post". Then show scheduled action status in a separate UI.